packages/scss/src/utilities/_spacing.scss

```diff
@@ -10,105 +10,105 @@
 /* Margin bottom */
 
```

Comment:
The horror! One-property-classes! Proudly bearing the hated `!important` prefix on top of that!
Is this really needed ?